### DISCUSSION TITLE
My header is screen width, but when a scroll bar appears it doesn't resize.

### DISCUSSION THREAD

**Discussion by dakamojo** [tailwindlabs/tailwindcss#11226] [Help] [answered]
My header looks like this...
```
<header class="flex w-screen h-12 bg-logo-blue text-white">
  <img src="/assets/logo.png" />
  <div class="px-3 flex items-center grow">Menu</div>
  <div class="px-3 flex items-center">xxx-xxx-xxxx</div>
</header>
```

It renders taking up the full with of the browser.  However once I add content to the page that is taller than the screen size, a vertical scroll bar appears as expected.  This scrollbar takes away from the screen size but the header doesn't resize.  So I end up with a horizontal scrollbar too that allows me to scroll the width of the vertical scrollbar.

What is the correct way to do this?

**Comment by wongjn** [comment, marked as answer]
You could consider not applying a `width` at all – `<header>` elements are block elements by default and would fill the width when displayed "naturally": https://play.tailwindcss.com/kXGLxFXVQl

**Reply by dakamojo** [reply]
Thanks that was precisely my problem.
